tests/compatibility-tests/src/main/resources/pageCounter/checkMessages.groovy

```diff
@@ -0,0 +1,31 @@
+package pageCounter
+
+import org.apache.activemq.artemis.api.core.SimpleString
+import org.apache.activemq.artemis.core.server.Queue
+import org.apache.activemq.artemis.tests.compatibility.GroovyRun
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
this is just a copy from the others groovy script

Reply:
Thats precisely why it needed fixed, so it doesnt get copied around further hehe :)

Reply:
I believe I did it this way because at the version where these were written years ago, it was mandatory to have the package statement on the first line.


I"m not 100% sure at this point if this was an actual issue or just my perception. I certainly remember having issues with the comment before the package.

But now since this is clear, I will change the others.

Reply:
@gemmellr  ^^